scripts/initialize.sh

```diff
@@ -72,9 +76,39 @@ while getopts "n:o:p:d:b:a:s:k:h" opt; do
 done
 shift $(expr $OPTIND - 1) # remove options from positional parameters
 
+CONFIG_DIR=${config_dir:-$CONFIG_DIR_DEFAULT}
+
+# read from the config file if a stake_owner is not provided as parameter
 if [ -z "$stake_owner" ]; then
-   echo 'Stake owner address must be provided. See --help'
-   exit 1
+   printf "\nReading stake owner address from the config file..."
+
+   # read from the config file when the stake owner is not provided
+   config_files=($CONFIG_DIR/*.toml)
+   config_files_count=${#config_files[@]}
+   while :; do
+      printf "\nSelect client config file: \n"
+      i=1
+      for o in "${config_files[@]}"; do
+         echo "$i) ${o##*/}"
+         let i++
+      done
+
+      read reply
+      if [ "$reply" -ge 1 ] && [ "$reply" -le $config_files_count ]; then
+         CONFIG_FILE_PATH=${config_files["$reply" - 1]}
+         break
+      else
+         printf "\nInvalid choice. Please choose an existing option number.\n"
+      fi
+   done
+
+   key_file_path=$(awk '/keyFile ?=(.*)$/{print $3}' $CONFIG_FILE_PATH | xargs)
+
+   printf "Reading an address from the key file: ${key_file_path}\n"
+
+   stake_owner=`cat $key_file_path | jq -jr .address`
+
```

Comment:
We may need need to ensure `0x` prefix is added to the address, as the key file doesn't have it.

```suggestion

[[ $stake_owner == 0x* ]] || stake_owner="0x$stake_owner"

```

Reply:
It works without `0x`, but I no harm adding [b06f99d](https://github.com/keep-network/keep-core/pull/3163/commits/b06f99d5366d843c555b80a89ebf6506b0a2cd83)